packages/seda-chain-contracts/src/data_request.rs

```diff
@@ -31,15 +35,27 @@ pub mod data_requests {
                 .flatten()
                 .is_some()
     }
-
+    #[allow(clippy::too_many_arguments)]
```

Comment:
With Franklin's comment this could be removed. :)

Reply:
When you rebase main, you should see `PostDataRequestArgs` in `msg.rs` for this